src/sentry/templatetags/sentry_assets.py

```diff
@@ -35,3 +35,12 @@ def crossorigin():
         # They share the same domain prefix, so we don't need CORS
         return ''
     return ' crossorigin="anonymous"'
+
+
+@register.simple_tag
+def locale_js_url(request):
+    lang_code = request.LANGUAGE_CODE
+    if lang_code in dict(settings.LANGUAGES):
```

Comment:
We should memoize this coersion from tuple of tuples to a dict since there's no reason it needs to be re-evaluated every time.

I'd suggest just doing something like:

```python
SUPPORTED_LANGUAGES = frozenset(settings.LANGUAGES.keys())
```

Inside of `src/sentry/conf/server.py`. Then you can just check `if lang_code in settings.SUPPORTED_LANGUAGES:` here.